syntax = "proto3";//声明了protobuf的版本

package  fixbug;//声明代码所在位置,对于c++来说是namespace

option cc_generic_services = true;

//定义登录请求基本类型 name pwd
message LoginRequest
{
	string name = 1;//数字表示第几个参数
	string pwd = 2;
}

message ResultCode
{
	int32 errCode = 1;
	string errMsg = 2;
}

message LoginResponse
{
	ResultCode result = 1;
	bool sucess  = 2;
}

service user
{
    rpc Login(LoginRequest) returns(LoginResponse);
}